source/documentation/deploying_services/postgresql.md.erb

```diff
@@ -585,20 +588,12 @@ When you use a high availability plan, Amazon RDS provides a hot standby service
 
 Refer to the [Amazon RDS documentation on the failover process](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.MultiAZ.html#Concepts.MultiAZ.Failover) for more information.
 
-You should test how your app deals with a failover to make sure you are benefiting from the high availability plan. Contact us at [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk) to arrange for us to trigger a failover for you.
+You should test how your app deals with a failover to make sure you are benefiting from the high availability plan.
```

Comment:
```suggestion
You should test how your app deals with a failover to make sure you are benefitting from the high availability plan.
```

Reply:
Add a link to our instructions on [forcing a failover](https://docs.cloud.service.gov.uk/deploying_services/postgresql/#force-a-failover)?

Reply:
Suggestion accepted.